wrench-app/src/main/java/com/izettle/wrench/database/migrations/Migrations.java

```diff
@@ -93,6 +93,16 @@ public void migrate(SupportSQLiteDatabase database) {
     public static final Migration MIGRATION_2_3 = new Migration(2, 3) {
         @Override
         public void migrate(SupportSQLiteDatabase database) {
+            {
+                // Reinstate indexes - due to a bug in a previous migration (1 -> 2) these indexes may be missing.
+                // This will recreate them in case they were missing so that migration can progress
+                database.execSQL("DROP INDEX IF EXISTS `index_configurationValue_configurationId_value_scope`");
+                database.execSQL("CREATE UNIQUE INDEX `index_configurationValue_configurationId_value_scope` ON `configurationValue` (`configurationId`, `value`, `scope`)");
```

Comment:
Where the heck does these names come from, and how would one validate them?

Reply:
They are auto generated by room and named by the fields included in the index. And verified by room to be existing and correct if you use their test tools.

Reply:
👍